src/pages/HomePage/HomePage.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react'
+import SubHeader from '../../components/SubHeader/SubHeader'
+import Carousel from '../../components/Carousel/Carousel'
+import categoryImage1 from '../../assets/Images/skincare-2.jpg'
+import categoryImage2 from '../../assets/Images/tampons.jpg'
+import categoryImage3 from '../../assets/Images/makeup-img-2.jpg'
+import categoryImage4 from '../../assets/Images/hair-care-img.jpg'
+import categoryImage5 from '../../assets/Images/body-care-img.jpg'
+import brandImage1 from '../../assets/Images/loreal_logo.jpg'
+import brandImage2 from '../../assets/Images/lakme_logo.jpg'
+import brandImage4 from '../../assets/Images/maybelline_logo.png'
+import brandImage5 from '../../assets/Images/Wishper_logo.png'
+import offerImage1 from '../../assets/Images/offerImage1.jpg'
+import offerImage2 from '../../assets/Images/offerImage2.jpg'
+import { Link } from 'react-router-dom'
+import './HomePage.css'
+import Footer from '../../components/Footer/Footer'
+
+const HomePage = () => {
+  return (
+    <>
+
+      <SubHeader />
+      <Carousel />
+      <h5 className='sub-heading'>Categories for you</h5>
+      <div class="featured-category-section flex-hz flex-wrap">
```

Comment:
try to see if you can or cannot make the code below into a component or try using an existing component and extending its styling through CSS, will make the homepage file better in terms of readability.

Reply:
Thanks for the review @ivinayakg . I am going to shift categories section to a seperate component after adding context.As of now I have kept that part static.